### PR TITLE
feat: add worker api mocks

### PR DESCRIPTION
**Pull request by meinenec**
Adds mocks for changes added in https://github.com/go-vela/server/pull/193

### CODE REVIEW

File: server/worker.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+			  "large:docker"
+			],
+			"active": true,
+			"last_checked_in": 1602612590
+		  },
+		{
+			"id": 2,
+			"hostname": "worker_2",
+			"address": "http://vela:8081",
+			"routes": [
```
Comment (kneal) [inline server/worker.go:52]: Might make sense to show these as a different dataset than the ones above

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Review by jbrockopp — APPROVED** [review]
LGTM

File: server/worker.go (hunk 1 above)
Comment (meinenec) [reply server/worker.go:52]: Can you clarify which you think should be in a separate dataset?

I'm not sure if you mean the routes or something else.

**Pull request merged**